src/main/kotlin/no/nav/syfo/melding/MeldingService.kt

```diff
@@ -47,9 +47,11 @@ class MeldingService(
         )
     }
 
-    private fun getBehandlerRefForConversation(conversationRef: UUID): UUID {
-        return database.getMeldingerInConversation(conversationRef)
-            .firstOrNull { !it.innkommende }
+    private fun getBehandlerRefForConversation(conversationRef: UUID, personIdent: PersonIdent): UUID {
+        return database.connection.use {
+            it.getUtgaendeMeldingerInConversation(conversationRef, personIdent)
+        }
+            .firstOrNull()
             ?.behandlerRef
             ?: throw IllegalStateException("Melding fra behandler mangler behandlerRef")
```

Comment:
Blir det riktigere å si at vi ikke fant behandlerRef for/å knytte til Melding fra behandler? Siden selve meldingen fra behandler aldri inneholder behandlerRef.

Reply:
Gjorde en endring nå 👌🏼